adapters/data_saver.py

```diff
@@ -1,33 +1,41 @@
 import datetime
 from abc import ABC, abstractmethod
-from typing import Optional
 
 from ports.data_parser import FileSaveError
+from jinja2 import Environment, FileSystemLoader
 
 
 class FileWriter(ABC):
 
-    def __init__(self, output_path: Optional[str] = None):
-        self.output_path = output_path
-
     @abstractmethod
     def write(self, content: str) -> None:
```

Comment:
Please update definition of `write` method

`def write(self, students_data: list[Student]) -> None:`